iati/core/schemas.py

```diff
@@ -24,15 +24,15 @@ class Schema(object):
 
     """
 
-    def __init__(self, name=None):
+    root_element_name = ''
```

Comment:
Is this the correct scope for the value?

If it's a constant, the name should be CAPITALISED.